service/build.gradle

```diff
@@ -50,7 +50,11 @@ dependencies {
 		implementation('org.json:json:20231013')
 	}
 
-	// Test deps
+    // TPS
+    implementation group: "bio.terra", name:"terra-policy-client", version:"1.0.5-SNAPSHOT"
```

Comment:
Is there a non-snapshot dependency we can pull in?

Reply:
There isn't unfortunately, not sure why but it seems it's a standard across our Java clients (see [Rawls deps](https://github.com/broadinstitute/rawls/blob/develop/project/Dependencies.scala#L129) for example)